Description: avoid privacy leaks in html code and documentation
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2021-07-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/README.md
+++ b/README.md
@@ -2,10 +2,6 @@
 
 Main project page: <https://emscripten.org>
 
-GitHub CI status: [![CircleCI](https://circleci.com/gh/emscripten-core/emscripten.svg?style=svg)](https://circleci.com/gh/emscripten-core/emscripten/tree/main)
-
-Chromium builder status: [emscripten-releases](https://ci.chromium.org/p/emscripten-releases)
-
 Overview
 --------
 
--- a/tools/toolchain_profiler.results_template.html
+++ b/tools/toolchain_profiler.results_template.html
@@ -31,7 +31,7 @@
 <html><head>
 <meta charset="UTF-8">
 <title>Emscripten toolchain profiler results</title>
-<script src="https://d3js.org/d3.v3.min.js"></script>
+<script src="/usr/share/javascript/d3/d3.min.js"></script>
 <style>
 .chart { shape-rendering: crispEdges; }
 .mini text { font: 9px sans-serif; }
--- a/site/source/_themes/emscripten_sphinx_rtd_theme/layout.html
+++ b/site/source/_themes/emscripten_sphinx_rtd_theme/layout.html
@@ -70,9 +70,6 @@
   {%- endblock %}
   {%- block extrahead %} {% endblock %}
 
-  {# Keep modernizr in head - http://modernizr.com/docs/#installing #}
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.6.2/modernizr.min.js"></script>
-
 </head>
 
 <body class="wy-body-for-nav" role="document">
@@ -134,7 +131,7 @@
 	('docs/index', 'GitHub', '')
 ] -%}
 
-   <a href="https://github.com/emscripten-core/emscripten"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
+   <a href="https://github.com/emscripten-core/emscripten">Fork me on GitHub</a>
   
    <div class="main-nav-bar" style="">   
    
